ansible/configs/kni-osp/env_vars.yml

```diff
@@ -0,0 +1,73 @@
+# sample vars configuration file
```

Comment:
I think we should move this all into `default_vars` rather than have 2 separate default vars files + the cloud provider specific one.

Reply:
wanted to separate "user" variables from "infrastructure definition" ones (or at least make it easy to create an "own_vars" to override those user variables), thought that would be preferable for this one to be able to get used as a starting point for other ospbmc based labs